pkg/plugin/volume.go

```diff
@@ -137,6 +137,11 @@ func ensurePVC(config map[string]string, log *logrus.Entry) error {
 		return nil
 	}
 
+	storageClassName := "default"
```

Comment:
I think this should be `nil`, because using the default storage class is not specifying this at all